x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/engine_creation_logic.test.ts

```diff
@@ -28,8 +29,79 @@ describe('EngineCreationLogic', () => {
     name: '',
     rawName: '',
     language: 'Universal',
+    isLoadingIndices: false,
+    indices: [],
+    indicesFormatted: [],
+    selectedIndex: '',
+    engineType: 'appSearch',
+    isSubmitDisabled: true,
   };
 
+  const mockElasticsearchIndices = [
```

Comment:
This is a minor nit and just mainly a note for the future. When these mock data elements get longer than a few lines, it's good to extract them into a `mock` file. Then the data can be re-used in other component tests as well.

Reply:
I agree, It didn't look that long in my taste. But I will move it out if I find an opportunity when integrating create button endpoint.